vehicles/src/modules/company-user-management/users/company-users.controller.ts

```diff
@@ -102,14 +103,41 @@ export class CompanyUsersController {
     return await this.userService.create(createUserDto, companyId, currentUser);
   }
 
+  /**
+   * A GET method defined with the @Get(':userGUID') decorator and a route of
+   * /companies/:companyId/users/:userGUID that retrieves a user details by its GUID.
+   *
+   * @param companyId The company Id.
+   * @param userGUID The GUID of the user.
+   *
+   * @returns The user details with response object {@link ReadUserDto}.
+   */
+  @ApiOkResponse({
+    description: 'The User Resource',
+    type: ReadUserDto,
+  })
+  @Roles(Role.READ_USER)
+  @Get(':userGUID')
+  async get(
+    @Req() _request: Request,
```

Comment:
```suggestion
    @Req() request: Request,
```

Reply:
for consistency - every other method in the controller uses `request: Request`

Reply:
Removed the field.